stdlib/email/src/main/ballerina/Ballerina.toml

```diff
@@ -0,0 +1,18 @@
+[project]
+org-name = "ballerina"
+version = "1.1.3"
```

Comment:
We need to update the version in all the places.

Reply:
Updated with version 1.1.5.